app/src/main/java/com/example/ana/exampleapp/Variables.java

```diff
@@ -17,14 +17,13 @@
  */
 public final class Variables {
     // The server IP
-    private static final String IP = "IP";
+    private static final String IP = "localhost";
     private static final String PORT = "27017";
     // MongoDB uri with the IP, authentication (user, password and mechanism), database, etc.
-    public static final String mongo_uri = "mongodb://androidUser:password@" + IP + ":" + PORT +
```

Comment:
I am ok with commenting this line as this will be  the case when developing, but please don't remove it as in a production environment authentication is needed. You can also added in a commit called something like **Adapt the project to work in the development environment**